### PR TITLE
API 27107 update model for AllergyIntolerance

### PR DESCRIPTION
**Pull request by bcastillomartel-va**
[API-27107](https://jira.devops.va.gov/browse/API-27107)

### CODE REVIEW

File: charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/VprGetPatientData.java [hunk 1]
```diff
@@ -247,6 +248,8 @@ public static class Results {
 
       @JacksonXmlProperty private Vitals vitals;
 
+      @JacksonXmlProperty private Reactions reactions;
```
Comment (jblefkove-va) [inline charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/VprGetPatientData.java:251]: Move `vitals` down so this list is alphabetical again

**Review by jblefkove-va — DISMISSED** [review]

File: charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/Reactions.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    @JacksonXmlElementWrapper(localName = "reactions")
+    List<NameAndVuidXmlAttribute> reaction;
+
+    @JacksonXmlProperty ValueOnlyXmlAttribute removed;
+
+    @JacksonXmlProperty ValueOnlyXmlAttribute severity;
+
+    @JacksonXmlProperty ValueOnlyXmlAttribute source;
+
+    @JacksonXmlProperty CodeAndNameXmlAttribute type;
```
Comment (jhulbert-va) [inline charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/Reactions.java:85]: Based on the PDF, this is value only.

Comment (bcastillomartel-va) [reply charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/Reactions.java:85]: I think the PDF might be incorrect. The example reactions had code and name `<type code='D' name='DRUG' />`

Comment (jblefkove-va) [reply charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/Reactions.java:85]: `VPR_XML_P7.xls` indicates `code` and `name` as well

File: charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/VprGetPatientData.java [hunk 3]
```diff
@@ -245,12 +246,14 @@ public static class Results {
 
       @JacksonXmlProperty private Meds meds;
 
-      @JacksonXmlProperty private Vitals vitals;
+      @JacksonXmlProperty private Reactions reactions;
```
Comment (jblefkove-va) [inline charon-models/src/main/java/gov/va/api/lighthouse/charon/models/vprgetpatientdata/VprGetPatientData.java:249]: Now move `reactions` down as well...

**Review by jhulbert-va — APPROVED** [review]

**Pull request merged**